tests/page/frame-evaluate.spec.ts

```diff
@@ -36,10 +36,13 @@ it('should have correct execution contexts', async ({ page, server }) => {
 });
 
 function expectContexts(pageImpl, count, browserName) {
-  if (browserName === 'chromium')
-    expect(pageImpl._delegate._mainFrameSession._contextIdToContext.size).toBe(count);
-  else
+  if (browserName === 'chromium') {
+    const contexts: Map<number, any> = pageImpl._delegate._mainFrameSession._contextIdToContext;
+    const notElectron = [...contexts.values()].filter(context => context._delegate._contextNameForTest !== 'Electron Isolated Context');
```

Comment:
You can just skip these tests in Electron and remove _contextNameForTest, since they seem to not test anything there.

Reply:
Given that electron creates its own isolated worlds, it seems like something could break here in a different way from stock chromium. I'd like to keep the tests.